src/main/java/org/launchcode/water_garden_tour/models/garden/Garden.java

```diff
@@ -10,6 +11,7 @@
 import java.util.List;
 
 @Entity
+@JsonIgnoreProperties({"hibernateLazyInitializer","handler"})
```

Comment:
This was added to facilitate the single garden map view. I tested and didn't see a negative impact to anything. This is a summary about it: To handle the problem related to the serialization of the model using Jackson API when the model attributes have a lazy loading defined, we have to tell the serializer to ignore the chain or helpful garbage that Hibernate adds to classes, so it can manage lazy loading of data by declaring @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"}) annotation.